packages/core/src/components/graphs/radar.ts

```diff
@@ -0,0 +1,463 @@
+
+		// Add event listeners
+		this.addEventListeners();
+
+		oldYScale = yScale; // save the current scale as the old one
+	}
+
+	// append temporarily the label to get the exact space that it occupies
+	getLabelDimensions = (label: string) => {
+		const tmpTick = DOMUtils.appendOrSelect(this.svg, `g.tmp-tick`);
+		const tmpTickText = DOMUtils.appendOrSelect(tmpTick, `text`).text(label);
+		const { width, height } = DOMUtils.getSVGElementSize(tmpTickText.node(), { useBBox: true });
+		tmpTick.remove();
+		return { width, height };
+	}
+
+	// Given a flat array of objects, if there are missing data on key,
+	// creates corresponding data with value = null
+	normalizeFlatData = (dataset: any) => {
+		const options = this.model.getOptions();
+		const { angle, value } = options.radar.axes;
+		const groupMapsTo = options.data.groupMapsTo;
+		const completeBlankData = Tools.flatMapDeep(this.uniqueKeys.map(key => {
+			return this.uniqueGroups.map(group => ({ [angle]: key, [groupMapsTo]: group, [value]: null }));
+		}));
+		return Tools.merge(completeBlankData, dataset);
+	}
+
+	// Given a a grouped array of objects, if there are missing data on key,
+	// creates corresponding data with value = null
+	normalizeGroupedData = (dataset: any) => {
+		const options = this.model.getOptions();
+		const { angle, value } = options.radar.axes;
+		const groupMapsTo = options.data.groupMapsTo;
+		return dataset.map(({ name, data }) => {
+			const completeBlankData = this.uniqueKeys.map(k => ({ [groupMapsTo]: name, [angle]: k, [value]: null }));
+			return { name, data: Tools.merge(completeBlankData, data) };
+		});
+	}
+
+	handleLegendOnHover = (event: CustomEvent) => {
+		const { hoveredElement } = event.detail;
+		const { opacity } = Configuration.options.radarChart.radar;
+		this.parent.selectAll("g.blobs path")
+			.transition(this.services.transitions.getTransition("legend-hover-blob"))
+			.style("fill-opacity", group => {
+				if (group.name !== hoveredElement.datum().name) {
+					return opacity.unselected;
+				}
+				return opacity.selected;
+			});
+	}
+
+	handleLegendMouseOut = (event: CustomEvent) => {
+		const { opacity } = Configuration.options.radarChart.radar;
+		this.parent.selectAll("g.blobs path")
+			.transition(this.services.transitions.getTransition("legend-mouseout-blob"))
+			.style("fill-opacity", opacity.selected);
+	}
+
+	destroy() {
+		// Remove event listeners
+		this.parent.selectAll(".x-axes-rect > rect")
+			.on("mouseover", null)
+			.on("mousemove", null)
+			.on("mouseout", null);
+		// Remove legend listeners
+		const eventsFragment = this.services.events;
+		eventsFragment.removeEventListener(Events.Legend.ITEM_HOVER, this.handleLegendOnHover);
+		eventsFragment.removeEventListener(Events.Legend.ITEM_MOUSEOUT, this.handleLegendMouseOut);
+	}
+
+	addEventListeners() {
+		const self = this;
+		const { angle, value } = this.model.getOptions().radar.axes;
```

Comment:
```suggestion
		const { angle, value } = Tools.getProperty(this.model.getOptions(), "radar", "axes");
```

also it seems `value` isnt being used